asciimatics/widgets/textbox.py

```diff
@@ -102,7 +104,7 @@ def update(self, frame_no):
         # if we have the input focus.
         if self._has_focus:
             line = str(display_text[display_line][0])
-            logger.debug("Cursor: %d,%d", display_start_column, display_column)
+#            logger.debug("Cursor: %d,%d", display_start_column, display_column)
```

Comment:
Was this a temporary change?  Would rather the log was reinstated (or totally deleted if it was problematic for some reason).

Reply:
Yeah, was temporary, was a bit noisy when I tested. I've put it back in, but happy to remove the line. Either way.